dev/Dockerfile-dev

```diff
@@ -1,6 +1,6 @@
 FROM python:3.8.5-slim-buster
 
-RUN apt-get update && apt-get install --yes make git curl ncat
+RUN apt-get update && apt-get install --yes make git curl ncat vim
```

Comment:
Debugging inside the lib in the container? ;)

Reply:
No, the static files like `setup.cfg` aren't mounted, but copied instead, and it bothered me that I had to restart the container for every change :D

Reply:
I would propose to mount them; but since they do not change often, it is ok to just install vim ;)

Reply:
We'd need to put them into a seperate directory then which we can mount and point all programs to this dir to look for the files instead of the base dir etc. It would mean a lot of restructuring